app/controllers/application_controller.rb

```diff
@@ -34,13 +34,17 @@ def adequate_scopes?
     required_scopes.all? { |scope| current_scopes.include?(scope) }
   end
 
+  def request_scopes
+    required_scopes | current_scopes
+  end
```

Comment:
Aha, so this is because the user isn't logged in. Any ideas on how to ask for the same scopes we have plus some more?